src/EventStore.Core/Settings/ThreadCountCalculator.cs

```diff
@@ -1,24 +1,55 @@
 using System;
+using EventStore.Common.Utils;
+using Serilog;
 
 namespace EventStore.Core.Settings {
 
 	public static class ThreadCountCalculator {
 		private const int ReaderThreadCountFloor = 4;
 
-		public static int CalculateReaderThreadCount(int configuredCount, int processorCount) {
+		public static int CalculateReaderThreadCount(int configuredCount, int processorCount,
+			bool isRunningInContainer) {
 			if (configuredCount > 0)
 				return configuredCount;
 
-			var readerCount = Math.Clamp(processorCount * 2, ReaderThreadCountFloor, 16);
+			if (isRunningInContainer) {
+				Log.Information(
```

Comment:
we used to log what value we are calculating in the case that the `configuredCount > 0` but we aren't any more, we should add that back please

Reply:
oh, I forgot to add that to the configured count > 0. I will add this.